src/main/java/uk/gov/companieshouse/api/accounts/utility/ixbrl/DocumentGeneratorCaller.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.companieshouse.api.accounts.utility.ixbrl;
+
+import org.springframework.stereotype.Component;
+
+/**
+ * Class to call the document generator to obtain the ixbrl location.
+ * Since the functionality has not been implemented yet, (STORY SFA-574), it returns an empty string.
+ *
+ * This class will change to call the new end point and it will return the s3 ixbrl location.
+ */
+@Component
+public class DocumentGeneratorCaller {
```

Comment:
Could this class implement an interface to make unit testing simpler?

Reply:
In my opinion, we should be adding an interface for one method only which returns an empty string.